CHANGELOG.md

```diff
@@ -8,9 +8,12 @@ Version number changes (major.minor.micro) in this package denote the following:
 - A major version will increase if there are any backwards-incompatible changes in any of the packages contained in this Docker image, or any other backwards-incompabile changes in the execution environment.
 
 ## [Unreleased]
+
+
+## [2.0.1] - 2017-03-10
 ### Changed
 - Various package version changes (#22)
-    - awscli 1.11.27 --> 1.11.60 
+    - awscli 1.11.27 --> 1.11.60
```

Comment:
this should probably be under "Fixed", right?  AWS packages didn't work in 2.0.0